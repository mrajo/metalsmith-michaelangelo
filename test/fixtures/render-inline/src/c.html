---
title: wut
template: other.html
---
Nothing happens