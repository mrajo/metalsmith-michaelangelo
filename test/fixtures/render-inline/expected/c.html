<h3>wut</h3>